src/Symfony/Bridge/Twig/Command/LintCommand.php

```diff
@@ -226,6 +240,21 @@ private function displayJson(OutputInterface $output, array $filesInfo): int
         return min($errors, 1);
     }
 
+    private function renderDeprecation(SymfonyStyle $output, Error $exception, ?string $file = null, ?GithubActionReporter $githubReporter = null): void
```

Comment:
Same here, I'm not sure there's a need for optional arguments as they seem always provided